compiler/src/iree/compiler/Preprocessing/Common/MakeSingleDispatchForFunction.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 The IREE Authors
+// Copyright 2023 The IREE Authors
```

Comment:
This shouldn't be needed.

Reply:
New file added needs to have the current year? Thats what I have been told, I actually dont know

Reply:
https://opensource.google/documentation/reference/copyright

> the year the file was created in full numeric form (e.g., 2010); don't change this if you edit, move, or copy the file.

If you are forking a file (and keeping most of the structure / code, not just grabbing the boilerplate), you can keep the original year. If the file is mostly new, you can put the current year (the year the file was authored, which might be earlier than the year it actually gets merged onto a shared branch).